forge-game/src/main/java/forge/game/card/CardFactory.java

```diff
@@ -124,62 +124,49 @@ private final static Card copySpellHost(final SpellAbility sourceSA, final Spell
         final Card source = sourceSA.getHostCard();
         final Card original = targetSA.getHostCard();
         final Game game = source.getGame();
-        final Card c = new Card(game.nextCardId(), original.getPaperCard(), game);
-        copyCopiableCharacteristics(original, c, sourceSA, targetSA);
+        int id = game.nextCardId();
```

Comment:
Wasn't the old method faster because we didn't have to rebuild the old card completely? (at least for single faced cards)

So while the last call is only done in simulation for tokens it means tests still use it so we might not see if it broke something? 🤔

Reply:
We don't "rebuild" the whole card for copySpellHost, but just enough so I can apply copy states

We need to add more tests for this one

Reply:
gonna merge this, tests can be added on a later PR